content/kubernetes/active-active/preview/edit-clusters.md

```diff
@@ -0,0 +1,193 @@
+---
+Title: Edit participating clusters for Active-Active database 
+linkTitle: Edit participating clusters
+description: Steps to add or remove a participating cluster to an existing Active-Active database with Redis Enterprise for Kubernetes.
+weight: 40
+alwaysopen: false
+categories: ["Platforms"]
+aliases: {
+/kubernetes/active-active/preview/add-cluster/,
+}
+---
+{{<banner-article bannerColor="#fff8dc">}}
+This feature is currently in public preview. Contact Redis support if you plan to use this feature.
+See [Create Active-Active databases for Kubernetes]({{<relref "/kubernetes/active-active/create-aa-database.md">}}) for the currently supported procedure.
+{{</banner-article>}}
+
+## Add a participating cluster
+
+Use the following steps to add a participating cluster to an existing Redis Enterprise Active-Active database (REAADB) for Kubernetes.
+
+### Prerequisites
+
+To prepare the Redis Enterprise cluster (REC) to participate in an Active-Active database, perform the following tasks from [Prepare participating clusters]({{<relref "/kubernetes/active-active/preview/prepare-clusters.md">}}):
+
+- Make sure the cluster meets the hardware and naming requirements. 
+- Enable the Active-Active controllers.
+- Configure external routing.
+- Configure `ValidatingWebhookConfiguration`.
+
+### Collect REC credentials
+
+To communicate with other clusters, all participating clusters need access to the admin credentials for all other clusters.
+
+1. Locate the file holding the admin credentials for all participating RECs created while [preparing the clusters]({{<relref "/kubernetes/active-active/preview/prepare-clusters.md">}}) (such as `all-rec-secrets.yaml`).
+
+1. Within that file, create a secret for the new participating cluster named `redis-enterprise-<rerc>`. 
+  {{<note>}}The file should contain a secret for each existing participating cluster.{{</note>}}
+
+    The example below shows a file (`all-rec-secrets.yaml`) holding secrets for two participating clusters:
+
+    ```yaml
+    apiVersion: v1
+    data:
+      password: 
+      username: 
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc1
+    type: Opaque
+
+    ---
+
+    apiVersion: v1
+    data:
+      password: 
+      username: 
+    kind: Secret
+    metadata:
+      name: redis-enterprise-rerc2
+    type: Opaque
+
+    ```
+
+1. Get the REC credentials secret for the new participating cluster.
+
+    ```sh
+    kubectl get secret -o yaml <rec-name>
+    ```
+
+    The admin credentials secret for an REC named `rec1` has this format:
+
+    ```yaml
+    apiVersion: v1
+    data:
+      password: ABcdef12345
+      username: GHij56789
+    kind: Secret
+    metadata:
+      name: rec1
+    type: Opaque
+    ```
+
+1. Add the username and password to the new secret for that REC and namespace.
+
+    This example shows the collected secrets file (`all-rec-secrets.yaml`) for `rec1` in namespace `ns1` and `rec2` in namespace `ns2`.
```

Comment:
also here it is no longer the convention of the rec name and ns.
suggestion:
This example shows the collected secrets file (`all-rec-secrets.yaml`) for `rerc1` that represents the `rec1` in namespace `ns1`, `rerc2` that represents the `rec2` in namespace `ns2` and `rerc3` that represents the `rec3` in namespace `ns3.

note that I have added rerc3 since we start here (in the previous steps) with example that shows 2 secrets we now add the 3rd to stay consisted.